torch/ao/quantization/_correct_bias.py

```diff
@@ -17,34 +17,38 @@
 _supported_modules_quantized = {nnq.Linear, nnq.Conv2d}
 
 def get_module(model, name):
-    ''' Given name of submodule, this function grabs the submodule from given model
-    '''
+    """Given name of submodule, this function grabs the submodule from given model."""
     return dict(model.named_modules())[name]
 
 def parent_child_names(name):
-    '''Splits full name of submodule into parent submodule's full name and submodule's name
-    '''
+    """Split full name of submodule into parent submodule's full name and submodule's name."""
     split_name = name.rsplit('.', 1)
     if len(split_name) == 1:
         return '', split_name[0]
     else:
         return split_name[0], split_name[1]
 
 def get_param(module, attr):
-    ''' Sometimes the weights/bias attribute gives you the raw tensor, but sometimes
+    """Get the parameter given a module and attribute.
+
+    Sometimes the weights/bias attribute gives you the raw tensor, but sometimes
     gives a function that will give you the raw tensor, this function takes care of that logic
-    '''
+    """
     param = getattr(module, attr, None)
     if callable(param):
         return param()
     else:
         return param
 
 class MeanShadowLogger(ns.Logger):
-    r"""A logger for a Shadow module whose purpose is to record the rolling mean
+    r"""Mean Logger for a Shadow module.
```

Comment:
Why does it needs to be a raw string?
```suggestion
    """Mean Logger for a Shadow module.
```